cached_network_image/example/lib/main.dart

```diff
@@ -1,6 +1,7 @@
 import 'package:cached_network_image/cached_network_image.dart';
 import 'package:flutter/material.dart';
 import 'package:baseflow_plugin_template/baseflow_plugin_template.dart';
+// ignore: depend_on_referenced_packages
```

Comment:
This shouldn't be ignored, the example should depend on it

Reply:
To fix this warning without this comment, we then need to specify flutter_blurhash to the pubspec. I wasn't sure that's what you'd like because the package is transitive dependency.  But, I can make an update per what's you'd like to have, just let me know.